doc/configuration.md

```diff
@@ -64,6 +64,16 @@ Parameters introduced by Snapclient:
 - `killall`: Kill all running librespot instances before launching librespot
 - `wd_timeout`: Restart librespot if it doesn't create log messages for x seconds
 
+#### Configuring 24 bits sample format
```

Comment:
I would rather do this implicitly: When the format is configurable in `stream_manager.cpp`, you can query the format in the constructor of `LibrespotStream` and do a validation of the sample rate and channels (must be 44100:X:2) and also validate the bits per sample: X to be valid (16, 24, 32) and add the `--format` parameter to the librespot `params_`

You can access the configured sampleformt within the `LibrespotStream` constructor with `sampleFormat_` and access the members `rate()`, `bits()` and `channels()`. Any invalid setting should throw an exception, so that Snapserver will exit and log the configuration error.